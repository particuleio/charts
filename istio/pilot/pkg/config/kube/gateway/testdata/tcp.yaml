apiVersion: networking.x-k8s.io/v1alpha1
kind: GatewayClass
metadata:
  name: istio
spec:
  controller: istio.io/gateway-controller
---
apiVersion: networking.x-k8s.io/v1alpha1
kind: Gateway
metadata:
  name: gateway
  namespace: default
spec:
  gatewayClassName: istio
  listeners:
  - port: 34000
    protocol: TCP
    routes:
      namespaces:
        from: All
      selector:
        matchLabels:
          selected: "yes"
      kind: TCPRoute
---
apiVersion: networking.x-k8s.io/v1alpha1
kind: TCPRoute
metadata:
  name: tcp
  namespace: default
  labels:
    selected: "yes"
spec:
  rules:
  - forwardTo:
    - serviceName: httpbin
      port: 9090
