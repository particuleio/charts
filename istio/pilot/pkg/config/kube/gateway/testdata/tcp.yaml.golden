apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  selector:
    istio: ingressgateway
  servers:
  - hosts:
    - '*'
    port:
      name: tcp-34000-gateway-gateway-default
      number: 34000
      protocol: TCP
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  creationTimestamp: null
  name: tcp-tcp-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - default/gateway-istio-autogenerated-k8s-gateway
  hosts:
  - '*'
  tcp:
  - route:
    - destination:
        host: httpbin.default.svc.domain.suffix
        port:
          number: 9090
---
